drivers/sdhc/sdhc_spi.c

```diff
@@ -631,13 +633,22 @@ static int sdhc_spi_request(const struct device *dev,
 				/* CMD12 is required after multiple read, or
 				 * to retry failed transfer
 				 */
-				sdhc_spi_send_cmd(dev,
+				stop_ret = sdhc_spi_send_cmd(dev,
 					(struct sdhc_command *)&stop_cmd,
 					false);
+				while ((stop_ret != 0) && (retries < 0)) {
```

Comment:
why is it retries < 0?

Reply:
Good catch, this should be `retries > 0`.